bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -184,7 +185,7 @@ private func dummy() {
     _ = NSLocalizedString("Make your content stand out by adding images, gifs, videos, and embedded media to your pages.", comment: "")
     _ = NSLocalizedString("Media options", comment: "translators: %s: block title e.g: \"Paragraph\".")
     _ = NSLocalizedString("Mention", comment: "")
-    _ = NSLocalizedString("Minimum height", comment: "")
```

Comment:
- Removal
- [Code reference](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/cover/edit.js#L581:L581)

This string is already translated as it's used in the web version, so we don't need to include it here.